_data/collaborators.yml

```diff
@@ -35,3 +35,14 @@
       link: https://www.linkedin.com/in/nehha/
     - name: github
       link: https://github.com/Neha
+
```

Comment:
I would just remove the extra empty line here to keep consistent with the rest :)

Reply:
I guess is normal for some editor to add empty line. We can configure that on our editor and agree how do we want to do. I am ok, with both ways